packages/binding.core/spec/usingBehaviors.ts

```diff
@@ -183,7 +186,7 @@ describe('Binding: Using', function () {
     testNode.innerHTML = "<div data-bind='using: someitem'>text" +
             "<!-- ko foreach: childprop --><span data-bind='text: $data'></span><!-- /ko --></div>"
 
-    var childprop = observableArray([])
+    var childprop: ObservableArray = observableArray([])
```

Comment:
weird that the `: ObservableArray` is needed, I'd expect `observableArray([])` to provide that

Reply:
Done, changed to generic in https://github.com/Auge19/tko/pull/4